File: @LV1/1.3/fizzbuzz/fizzbuzz.cpp
#include <cstdio>

int main() {
    int x, y, n;
    scanf("%d %d %d", &x, &y, &n);

    for (int i = 1; i <= n; i++) {
        int mx = i % x == 0;
        int my = i % y == 0;

        if (mx && my) {
            puts("FizzBuzz");
        } else if (mx) {
            puts("Fizz");
        } else if (my) {
            puts("Buzz");
        } else {
            printf("%d\n", i);
        }
    }
}

File: @LV1/1.3/fizzbuzz/fizzbuzz.go
package main

import "fmt"

func fizzbuzz(x int, y int, n int) string {
	mx := n%x == 0
	my := n%y == 0

	if mx && my {
		return "FizzBuzz"
	} else if mx {
		return "Fizz"
	} else if my {
		return "Buzz"
	} else {
		return fmt.Sprintf("%d", n)
	}
}

func main() {
	var x, y, n int

	fmt.Scanf("%d %d %d", &x, &y, &n)

	for i := 1; i <= n; i++ {
		fmt.Println(fizzbuzz(x, y, i))
	}
}
